README.md

```diff
@@ -47,6 +47,15 @@ An example for such a "porcelain" API would be [ROS 2](https://www.ros.org/). Ot
 
 You can find the full API documentation on 🌐 [https://iceoryx.io](https://iceoryx.io).
 
+### Supported Platforms
+
+|Operating System| supports access rights for shared memory | command line parsing    |
+|----------------|:----------------------------------------:|:-----------------------:|
+| Linux          | X                                        | X                       |
```

Comment:
The `x` feels a bit ambiguous. Can we have 👍 and 💩?

Reply:
I added green check marks, red crosses and a black check to make it more clear.

Reply:
I'd like the :hankey: more but one cannot get everything :laughing:

Reply:
@elBoberido I know how much poop means to you and therefore I will start collecting money from all our developers so that we can buy you this wonderful face mask:

https://images-na.ssl-images-amazon.com/images/I/61wrOUWncYL._AC_UX679_.jpg